src/sass/_theme_rst.sass

```diff
@@ -270,6 +270,12 @@
   .hlist
     width: 100%
 
+  // Global classifier class format
+  // the classifier class is used both inside the Html4 writer tables and the html5 writer definition lists
+  span.classifier
```

Comment:
This could be too generic for user content. I could see user content like API docs (similar to the demo docs API docs) having overlap on this selector. Perhaps a bit more specificity here would be an okay middle ground, ie `dl > dt > span.classifier`